packages/deployment-cli/package.json

```diff
@@ -26,11 +27,12 @@
     "web3.storage": "^3.5.4"
   },
   "devDependencies": {
-    "@babel/plugin-transform-runtime": "^7.17.0",
-    "babel-eslint": "10.1.0",
-    "prettier": "2.6.1",
-    "tslint": "^6.1.3",
-    "tslint-config-prettier": "^1.18.0"
+    "@typescript-eslint/eslint-plugin": "^5.35.1",
```

Comment:
@navFooh if i remember correctly, `@babel/plugin-transform-runtime` was required to run this script properly - although, i think it should be a regular `dependency` rather than `devDependency`

The list of plugins are here: https://github.com/glifio/modules/blob/actor-utils/packages/deployment-cli/.babelrc#L4

Although, i could've sworn @babel/preset-env also included the runtime helpers... That's another rabbit hole, which I think could be in a different PR (clean up the .babelrc)

Reply:
Hmm yeah there is a `package.json` file in the root folder that does have the `@babel/plugin-transform-runtime`. But I'm not sure how that even works, are those shared between the modules?

Reply:
It should be able to find it if it's up there in the root